### PR TITLE
warc-embed > wacz-exhibitor

### PR DESCRIPTION
**Pull request by matteocargnelutti**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <!doctype html>
 <html lang="en">
   <head>
-    <title>My WARC Museum 🏛️ | WARC EMBED Demo</title>
+    <title>My Web Archives Museum 🏛️ | Demo</title>
```
Comment (bensteinberg) [inline public/index.html]:
```suggestion
    <title>My Web Archive Museum 🏛️ | Demo</title>
```

File: public/index.html [hunk 2]
```diff
@@ -12,7 +12,7 @@
   <body id="index">
 
     <main>
-      <h1><span aria-hidden="true">🏛️<br></span>My WARC Museum</h1>
+      <h1><span aria-hidden="true">🏛️<br></span>My Web Archives Museum</h1>
```
Comment (bensteinberg) [inline public/index.html:15]:
```suggestion
      <h1><span aria-hidden="true">🏛️<br></span>My Web Archive Museum</h1>
```

File: README.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# warc-embed-demo 🏛️
-Tech demo for [`warc-embed`](https://github.com/harvard-lil/warc-embed). 
+# wacz-exhibitor-demo 🏛️
+Tech demo for [Wacz-Exhibitor](https://github.com/harvard-lil/wacz-exhibitor).
```
Comment (bensteinberg) [inline README.md]:
```suggestion
Tech demo for [WACZ-Exhibitor](https://github.com/harvard-lil/wacz-exhibitor). 
```

**Review by bensteinberg — APPROVED** [review]
I am making two suggestions here in three changes -- no pressure on either of these, but my copyediting eye snagged on both of them. WRT the new name for warc-embed, I think if you don't want to capitalize WACZ, it should be wacz not Wacz. If it's wacz, then the next word should probably be exhibitor not Exhibitor. (And I'm not sure the name needs to inherit the hyphen from the repo name.)

**Comment by matteocargnelutti** [comment]
Thanks @bensteinberg !

**Pull request merged**